build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -200,6 +201,7 @@ private static bool CheckIfAllBuildsHavePublished()
                  { "sharedfx_Ubuntu_16_10_x64", false },
                  { "sharedfx_RHEL_x64", false },
                  { "sharedfx_OSX_x64", false },
+                 { "sharedfx_Debian_8_armel", false },
```

Comment:
Please comment this out as this needs to be enabled when pipeline build is enabled for armel.